doc/vagrant/README.md

```diff
@@ -0,0 +1,29 @@
+# Trying out Elektra with Vagrant
+This Vagrantfile provisions a machine based on debian/jessie64 with all tools to build Elektra.
+
+If [Vagrant](https://www.vagrantup.com/) is installed on your machine change to the directory containing the file you are currently reading and start the VM with
+```bash
+$ vagrant up
+```
+When you run this command for the first time this will take some time as Vagrant will install all required packages.
+
+When the machine is ready and running, access it with
+```bash
+$ vagrant ssh
+```
+In this SSH session you can interact with the machine.
+
+You could for instance change to `/vagrant`, which contains the contents of `share` from the host machine.
+
+Therefore it should contain an exemplary shell script that pulls a specified snapshot of Elektra from Github, installs it in the VM and builds a .deb package from it.
```

Comment:
Yes, it does some temporary install to track everything, but why mention this in the README.md? Important for the reader of the readme is that after running the shell script, Elektra will be installed (maybe unless some command-line option is provided?)

Thus I would not mention the temporary install (implementation detail of checkinstall), but I would mention that the deb gets installed.